docs/sources/http_api/licensing.md

```diff
@@ -7,6 +7,39 @@ aliases = ["/docs/grafana/next/http_api/licensing/"]
 
 > Licensing is only available in Grafana Enterprise. Read more about [Grafana Enterprise]({{< relref "../enterprise" >}}).
 
+## Check license availability
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+`GET /api/licensing/check`
+
+Checks if a valid license is available.
+
+**Example request:**
+
+```http
+GET /api/licensing/check
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+
+{}
+```
+
+**Example response:**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 4
+
+{"true"}
```

Comment:
Isn't the output just `true`? Otherwise, this is not valid JSON (as a JSON object needs both keys and values)

```suggestion
true
```

Reply:
Oh right! Thanks @sakjur